docs/orchestrating-elastic-stack-applications/kibana.asciidoc

```diff
@@ -206,6 +206,18 @@ To deploy more than one instance of Kibana, all the instances must share a same
 * `xpack.reporting.encryptionKey`
 * `xpack.encryptedSavedObjects.encryptionKey`
 
+[TIP]
+====
+If you need to access these encryption keys, you can find them using the `kubectl get secrets` command. For example, link:{kibana-ref}/xpack-security-secure-saved-objects.html#encryption-key-rotation[rotating encryption keys] extract the current value of `xpack.encryptedSavedObjects.encryptionKey` in order to set it as an historical encryption value under `xpack.encryptedSavedObjects.keyRotation.decryptionOnlyKeys`.
```

Comment:
<img width="852" alt="Screenshot 2024-10-24 at 09 33 56" src="https://github.com/user-attachments/assets/e881f396-4a75-4e00-b31c-537b91b4dcd0">

The "For example" sentence is kind of broken here. 

Also I believe we should not try to squeeze the key rotation topic into this `TIP`. Because just doing what you are describing here will not rotate the encryption keys it will just duplicate the key into a decpryption-only key and lead to a crash looping Kibana pod, because the primary encryption key cannot also be a decryption-only key. So the primary key has also to be changed to a new valid (!) value. I believe key rotation is something ECK should orchestrate not users manually. This would be a new operator feature. 

```suggestion
If you need to access these encryption keys, you can find them using the `kubectl get secrets` command. For example, link:{kibana-ref}/xpack-security-secure-saved-objects.html#encryption-key-rotation[to roate the encryption keys] extract the current value of `xpack.encryptedSavedObjects.encryptionKey` in order to set it as a decryption-only key under `xpack.encryptedSavedObjects.keyRotation.decryptionOnlyKeys`.
```